CMakeLists.txt

```diff
@@ -80,4 +80,7 @@ target_include_directories(moonlight-common-c PRIVATE
   ${CMAKE_CURRENT_SOURCE_DIR}/reedsolomon
 )
 
-target_compile_definitions(moonlight-common-c PRIVATE HAS_SOCKLEN_T)
\ No newline at end of file
+target_compile_definitions(moonlight-common-c PRIVATE HAS_SOCKLEN_T)
```

Comment:
This stuff will obviously need to be reverted before merge

Reply:
Sorry, yes : a little mess with my submodules and pushed branch